components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -3035,4 +3035,19 @@ public static void resetAuthenticationContext(AuthenticationContext context) {
         context.setRetrying(false);
         context.setCurrentAuthenticator(null);
     }
+
+    /**
+     * Check whether the JIT provisioning enhanced feature is enabled.
+     *
+     * @return true if the JIT provisioning enhanced features is enabled else return false.
+     */
+    public static boolean isJitProvisionEnhancedFeature() {
```

Comment:
`isJitProvisionEnhancedFeatureEnabled` is more meaningful IMO. WDYT?

Reply:
Also, let's use the same FrameworkUtils method for scim as well for the PR: https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/367

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3583/commits/34412bacbfef7ae189478c850a9b3795a05e9944